### ISSUE TITLE
Как сделать модальное окно в списке.

### ISSUE THREAD

**Issue by PSalador** [tabuna/Vision-Clinic#1]
Например в Vision Clinic
Когда заходишь в карточку клиента (Patient card) там два списка Appointments и Invoices.
В верхнем меню можно через модальное окно добавить Appointment, а как сделать чтобы можно было редактировать через модальное окно?

**Comment by tabuna** [comment]
Модальное окно отдаёт [Layouts](https://github.com/tabuna/Vision-Clinic/blob/master/app/Http/Screens/Clinic/Patient/PatientEdit.php#L94-L98)

```php
            Layouts::modals([
                'Appointments' => [
                    Appointment::class,
                ],
            ]),
```

В этом [Layouts](https://github.com/tabuna/Vision-Clinic/blob/master/app/Http/Layouts/Clinic/Patient/Appointment.php) можно определить какие данные брать из query 


Например замени 

```php
            'appointment_time' => [
                'tag'      => 'datetime',
                'name'     => 'appointment_time',
                'required' => true,
                'title'    => 'Time',
            ],
```
на 

```php
           'first_name' => [
                'tag'      => 'input',
                'type'     => 'text',
                'name'     => 'patient.first_name',
                'max'      => 255,
                'required' => true,
                'title'    => 'First name',
            ],
```

И имя его уже будет заранее вписано в поле для ввода.

**Comment by PSalador** [comment]
Нет, я про другое. У тебя не реализовано изменение  Appointment (Назначений пациенту), то есть добавить можно, а изменить нет. При том, что этих Appointments может быть 10 или больше. Нужно делать под каждое назначение пациету -отдельное модальное окно или как-то скриптом подгружать?

**Comment by tabuna** [comment]
> Нет, я про другое. У тебя не реализовано изменение Appointment (Назначений пациенту), то есть добавить можно, а изменить нет.

Пример не полный

> Притом, что этих Appointments может быть 10 или больше. Нужно делать под каждое назначение пациету -отдельное модальное окно или как-то скриптом подгружать?

Под каждый конечно же не нужно будет. Просто отдельный экран
